modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -152,7 +152,7 @@ protected static TestValue value(BinaryRow binaryRow) {
         return new IgniteBiTuple<>(key(binaryRow), value(binaryRow));
     }
 
-    protected static List<IgniteBiTuple<TestKey, TestValue>> toList(Cursor<ReadResult> cursor) throws Exception {
+    protected static List<IgniteBiTuple<TestKey, TestValue>> drainToList(Cursor<ReadResult> cursor) throws Exception {
```

Comment:
Why "drain"? method does not throw `Exception`.

Reply:
Removed the `throws` declaration.

I renamed the method so that we can statically import `Collectors.toList`.

There is `BlockingQueue.drainTo()` that drains the queue to a collection. `drainToList` follows the same logic.

Reply:
Ok